spec/ios_update_release_notes_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe Fastlane::Actions::IosUpdateReleaseNotesAction do
+  let(:release_notes_txt) { File.join(File.dirname(__FILE__), 'RELEASE-NOTES.txt') }
+  let(:changelog_md) { File.join(File.dirname(__FILE__), 'CHANGELOG.md') }
+
+  after do
+    FileUtils.rm([release_notes_txt, changelog_md], force: true)
+  end
```

Comment:
Same, please don't make running the test result in creating files in the repo itself.

This is especially problematic if your tests happen to crash and the files would not be deleted, unlike with using `Dir.mktmpdir` or `in_tmp_dir` or `with_tmp_file`.

Reply:
Done in 6baf51dbd13b8667d9503cb60c63ad399b19f47b and 9d4b18a2ff1b408150c79588a89dd315392a6956